lib/thesis/stores/ecto_store.ex

```diff
@@ -103,6 +123,12 @@ defmodule Thesis.EctoStore do
     repo.insert_or_update!(page_changeset)
   end
 
+  defp backup_page(page, page_contents) do
+    serialized_page_data = prepare_page_backup_data(page, page_contents)
+    save_backup(page.id, serialized_page_data)
+    :ok
```

Comment:
I don't think we should return `:ok` every time here. This should return the result of `save_backup/2`, or check its result and return an `{:ok, backup}` or `{:error, error}` tuple.